src/installation/system_requirements.md

```diff
@@ -17,23 +17,17 @@ For the Epinio server, and related deployments we recommend to consider the foll
 
 In addition, extensive requirements for your workload (apps) would add to that.
 
-A default storage class (with annotation `storageclass.kubernetes.io/is-default-class: "true"`) is needed.
-
-While Kubernetes 1.22 is supported there is an issue when the container runtime is `containerd > 1.5.6`.
-The [pack cli](https://github.com/buildpacks/pack) is placing too much information into the
-image layers. The relevant issue is: https://github.com/paketo-buildpacks/full-builder/issues/415
+### Storage Class
 
+A default storage class (with annotation `storageclass.kubernetes.io/is-default-class: "true"`) is needed.
```

Comment:
Maybe we can give a link on how to do that or a command like:
`kubectl patch storageclass <name> -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'`

Reply:
We shouldn't have too much kube commands within our documentation. I think it's described well enough, and it can be found easily, on how to make the SC default (which is also widely the default in the meantime).